src/cascadia/TerminalApp/Pane.cpp

```diff
@@ -793,20 +793,24 @@ void Pane::_SetupChildCloseHandlers()
 // - rootSize: The dimensions in pixels that this pane (and its children should consume.)
 // Return Value:
 // - <none>
-void Pane::_CreateRowColDefinitions(const Size& rootSize)
+void Pane::_CreateRowColDefinitions(const Size& /*rootSize*/)
 {
+    auto first = _desiredSplitPosition * 100.0f;
+    auto second = 100.0f - first;
     if (_splitState == SplitState::Vertical)
     {
         _root.ColumnDefinitions().Clear();
 
         // Create two columns in this grid: one for each pane
-        const auto paneSizes = _CalcChildrenSizes(rootSize.Width);
+        // const auto paneSizes = _CalcChildrenSizes(rootSize.Width);
 
         auto firstColDef = Controls::ColumnDefinition();
-        firstColDef.Width(GridLengthHelper::FromValueAndType(paneSizes.first, GridUnitType::Star));
+        // firstColDef.Width(GridLengthHelper::FromValueAndType(paneSizes.first, GridUnitType::Star));
+        firstColDef.Width(GridLengthHelper::FromValueAndType(first, GridUnitType::Star));
 
         auto secondColDef = Controls::ColumnDefinition();
-        secondColDef.Width(GridLengthHelper::FromValueAndType(paneSizes.second, GridUnitType::Star));
+        // secondColDef.Width(GridLengthHelper::FromValueAndType(paneSizes.second, GridUnitType::Star));

```

Comment:
comments

Reply:
gah the last commit didn't get pushed ;__;